<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionAdaptationToolV2HandlerDialog</class>
 <widget class="QDialog" name="MotionAdaptationToolV2HandlerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MotionAdaptationToolV2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ChooseMotionGroupBox">
     <property name="title">
      <string>Select objects Example</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="ChooseMotionComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Transformation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QSpinBox" name="xOffset">
           <property name="minimum">
            <number>-500</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>y offset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="yOffset">
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="zOffset">
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>x offset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>z offset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="randomButton">
     <property name="text">
      <string>Random init</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="dButton">
     <property name="text">
      <string>DynamicOffset</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="storeButton">
     <property name="text">
      <string>Store motion</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
